### ISSUE TITLE
python-jl cannot find the module pytest

### ISSUE THREAD

**Issue by Laxzal** [JuliaPy/pyjulia#517]
Hi,

I am using PyCharm for PyTest and I am not sure if this is correct to do but I set the env interpreter to python-jl. Now, when I attempt to run a test, it informs me the module PyTest cannot be found. I do have PyTest installed and when I set the env interpreter to just python, the test runs smoothly.

Is this expected to happen?

**Comment by mkitti** [comment]
Which Python environment is python-jl using?

**Comment by Laxzal** [comment]
If I understand your question correctly, I am using mamba/conda environment

**Comment by mkitti** [comment]
No, I mean what specific conda environment is python-jl using? Or where is it located?

It may be using a different environment than the one you are in. In fact, this is the default behavior. If you want to make it use your current conda environment, you will need to tell Julia about that unless you installed julia via conda.
